topdown/http.go

```diff
@@ -110,6 +114,15 @@ func executeHTTPRequest(bctx BuiltinContext, obj ast.Object) (ast.Value, error)
 		}
 	}
 
+	// check if redirects are enabled
+	if enableRedirect {
```

Comment:
[nit] We can of course be explicit, but I think this section of code is equivalent to
```go
if !enableRedirect {
	client.CheckRedirect = func(*http.Request, []*http.Request) error {
		return http.ErrUseLastResponse
	}
}
```